src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+// Actions
+
+const ADDBOOK = 'books/ADDBOOK';
+const REMOVEBOOK = 'books/DELETE';
+const books = [];
+
+// Reducers
+export default function reducerBook(state = books, action = {}) {
+  switch (action.type) {
+    case ADDBOOK:
+      return [
+        ...books, action.book,
+      ];
+    case REMOVEBOOK:
+      return [
+        books.map((book) => book.id !== action.id),
+      ];
+    default:
+      return state;
+  }
+}
+
+// Action creators
+export function addBook(book) {
+  return { type: ADDBOOK, book };
+}
```

Comment:
- [ ] You have implemented the Book actions and reducers. Well done. However, when I attempt to add a new book, the page reloads and no book is added. The reason is that you simply created the `addBook()` action but you are not dispatching it yet.